tests/integration/components/bs-modal-test.js

```diff
@@ -228,6 +228,89 @@ test('when modal has a form and the submit button is clicked, the form is submit
   this.$('.modal .modal-footer button[type=submit]').click();
 });
 
+test('modal element is focused when no [autofocus] element is present and fade=false', function (assert) {
```

Comment:
I think the first two tests ("modal element is focused ...") are not needed, even a bit contra productive in the sense that you should write test for public interfaces and not (private) implementation details. Focusing the modal is an implementation detail, the feature is that pressing the escape key closes the modal. This should be testet, and this test is in place already. So just drop these two tests.

However another test that checks if closing the modal with the escape key works when an input element is autofocused would be nice!